src/Snakemake/rules/QC/picard.smk

```diff
@@ -0,0 +1,60 @@
+localrules:
+    touchBatch,
+
+
+rule picardHsMetrics:
+    input:
+        bam="DNA_bam/{sample}-ready.bam",
+        intervals=config["bed"]["intervals"],
+    output:
+        "qc/{sample}/{sample}.HsMetrics.txt",
+    log:
+        "logs/qc/picard/HsMetrics/{sample}.log",
+    singularity:
+        config["singularity"]["picard"]
```

Comment:
maybe change to 
config["singularity"].get("picard", config["singularity"].get("default", "")) and add
```
- picard=2.25.0
```
 to  https://github.com/clinical-genomics-uppsala/Twist_DNA/blob/develop/dockerfiles/somatic/environment.yml 

and update https://github.com/clinical-genomics-uppsala/Twist_DNA/blob/develop/dockerfiles/somatic/VERSION with new version (0.0.15)

Reply:
done